usr/lib/common/obj_mgr.c

```diff
@@ -749,16 +749,25 @@ CK_RV object_mgr_create_final(STDLL_TokData_t *tokdata,
     return rc;
 }
 
-/* destroy_object_cb
- *
- * Callback used to delete an object from the object map btree and whichever
- * other btree its in (based on its type)
- */
-void destroy_object_cb(STDLL_TokData_t *tokdata, void *node)
+CK_RV object_mgr_destroy_object(STDLL_TokData_t *tokdata,
+                                SESSION *sess, CK_OBJECT_HANDLE handle)
 {
-    OBJECT_MAP *map = (OBJECT_MAP *) node;
+    CK_RV rc = CKR_OK;
+    OBJECT_MAP *map;
     OBJECT *o;
 
+    if (!sess) {
```

Comment:
looks like sess param has been and still is unused except for checking != NULL. Why not remove it ?

Reply:
Good catch. 
I will leave the parameter there for consistency, since most other obj_mgr_xxx functions also get a session as parameter, but I remove the NULL check and mark it as UNUSED instead.